presto-main/src/main/java/com/facebook/presto/sql/planner/LocalExecutionPlanner.java

```diff
@@ -2323,7 +2324,8 @@ private PhysicalOperation createLookupJoin(JoinNode node,
                     dynamicFilter.getTupleDomainConsumer(),
                     filterBuildChannels,
                     getDynamicFilteringMaxPerDriverRowCount(context.getSession()),
-                    getDynamicFilteringMaxPerDriverSize(context.getSession()));
+                    getDynamicFilteringMaxPerDriverSize(context.getSession()),
+                    getDynamicFilteringRangeRowLimitPerDriver(context.getSession()));
         }
 
         private Optional<LocalDynamicFilter> createDynamicFilter(PhysicalOperation buildSource, AbstractJoinNode node, LocalExecutionPlanContext context, int partitionCount)
```

Comment:
can we add the `getDynamicFilteringRangeRowLimitPerDriver(context.getSession())));` for `new DynamicFilterSourceOperatorFactory` in `createDynamicFilter ` as well

Reply:
IIUC, `DynamicFilterSourceOperatorFactory` c-tor is only called at `createDynamicFilterSourceOperatorFactory`.
I am not sure if I understood the suggestion... did you mean to add min/max support to `LocalDynamicFilter`?

Reply:
sorry, my bad, was looking at the wrong line